janusgraph-cql/src/test/java/org/janusgraph/diskstorage/cql/CQLDistributedStoreManagerTest.java

```diff
@@ -54,6 +55,7 @@ public void tearDown() throws BackendException {
     @Override
     @Test
     @FeatureFlag(feature = JanusGraphFeature.OrderedScan)
+    @Disabled(value = "Test can't run inside of a container")
```

Comment:
I didn't check it but could you maybe give a short explanation why can't we execute this test?
As I understand we can execute it in TravisCI, so why can't we execute it in GitHub Actions?

Reply:
I tried to make the skip description independently of GitHub Actions. The test isn't build to run inside of container which is normal case in GitHub Actions and not on the TravisCI side.

Reply:
Do you have an error log of this test? I just want to understand why it can't be executed inside a container. What are the limitations of it? Is it related to networks? GitHub Actions allow to configure networks. I'm not sure I understand why this test can't be executed inside the container. Does it have some specific requirements which are not accessible inside a container?

Reply:
It will return remote instead of local. The reason is testcontainers fallback to the gateway if testcontainers is running inside a container. This leads to a non localhost address.